psi4/src/export_mints.cc

```diff
@@ -1067,6 +1069,8 @@ void export_mints(py::module& m) {
              "Gradient of MO basis OEI integrals: returns (3 * natoms) matrices")
         .def("mo_oei_deriv2", &MintsHelper::mo_oei_deriv2,
              "Hessian  of MO basis OEI integrals: returns (3 * natoms)^2 matrices")
+        .def("mo_overlap_half_deriv1", &MintsHelper::mo_overlap_half_deriv1,
+             "Half-derivative of MO basis overlap integrals: returns (3 * natoms) matrices")
```

Comment:
consider naming vars. gives user option of remembering var names rather than order. `"side"_a, "atom"_a, "C1"_a, "C2"_a`

Reply:
Done. For this instance and surrounding ones.